api/diffractionimage.py

```diff
@@ -47,9 +47,9 @@ def __init__(self):
     def get(self):
         args = self.reqparse.parse_args()
 
-        h5exts = ["h5", "nxs"]
+        h5exts = {"h5", "nxs"}
```

Comment:
Out of curiosity what's the thinking behind this change?
Is it just to ensure a unique set of extensions?
Seems somewhat unnecessary with a simple, hard coded list and not related to the main point of this changeset...